pytest-embedded-idf/pytest_embedded_idf/unity_tester.py

```diff
@@ -392,9 +392,7 @@ def _run_multi_stage_case(
             self._get_ready(timeout)
             self.confirm_write(case.index, expect_str=f'Running {case.name}...')
 
-            # here we can't use confirm_write because the sub cases won't print anything
-            # can't rely on time.sleep either cause it will break some time-related child cases
-            self.write(str(sub_case['index']))
+            self.confirm_write(str(sub_case['index']), expect_str=f"Running stage {sub_case['index']}...")
```

Comment:
this might be a breaking change on the esp-idf release branches. Let's update it in 2.x.

Reply:
Or after backport the subcase print in all release branches.

Reply:
Let's create another PR for these changes, and in this PR, we can disable this testcase?

Reply:
Sure I agree to move these to another PR.